ext/rugged/rugged_tree.c

```diff
@@ -489,6 +489,8 @@ void rugged_parse_merge_options(git_merge_options *opts, VALUE rb_options)
 
 		if (RTEST(rb_hash_aref(rb_options, CSTR2SYM("renames")))) {
 			opts->flags |= GIT_MERGE_FIND_RENAMES;
+		} else {
+			opts->flags &= ~GIT_MERGE_FIND_RENAMES;
```

Comment:
You could drop the first test, since this is set already, and just `if (!RTEST...)` and unset that bit.